mcp/kubev2v/kubevirt_server.py

```
@@ -0,0 +1,2146 @@
+        if not service_name:
+            service_name = f"{resource_name}-service"
+        
+        kubectl_args = ["delete", "service", service_name]
+        if namespace:
+            kubectl_args.extend(["-n", namespace])
+        
+        try:
+            result = subprocess.run(["kubectl"] + kubectl_args, capture_output=True, text=True, check=True, timeout=120)
+            return f"Service unexpose completed: {result.stdout.strip()}"
+        except subprocess.CalledProcessError as e:
+            raise VirtctlError(f"Failed to unexpose service: {e.stderr}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)